src/containers/shared/components/DomainLink.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { decodeHex } from '../transactionUtils'
+
+interface Props {
+  className?: string
+  decode?: boolean
+  domain: string
+}
+
+const DomainLink = (props: Props) => {
+  const { className, decode, domain } = props
+
+  return (
+    <a
+      className={className}
```

Comment:
I would default the className of "domain" and use className to add additional classes.  `clx` is a good tool for that.

Reply:
Aren't there some places (like src/containers/Token/TokenHeader/index.js) where we don't want className="domain"?